### PR TITLE
add an assume-yes flag to generate

### PR DESCRIPTION
**Pull request by mightyguava**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,8 @@ The output will contain 3 `CERTIFICATE` blocks instead of a `CERTIFICATE_REQUEST
 
 Add the `--no-csr` flag to skip the certificate signing request being printed. In this case, you will not be prompted to touch your Yubikey.  
 This option is useful if you don't need the generated key to be a part of an existing PKI.  
-you can still verify the key's certificate using Yubico's certificate [here](https://developers.yubico.com/PIV/Introduction/PIV_attestation.html)
+you can still verify the key's certificate using Yubico's certificate [here](https://developers.yubico.com/PIV/Introduction/PIV_attestation.html).
+The `--assume-yes` flag can be used in combination with the `--no-csr` option to disable its y/n prompt.
```
Comment (linuxwolf) [inline README.md:53]: I think you mean `--self-sign` here?

**Review by linuxwolf — APPROVED** [review]
LGTM with a nit

File: README.md (hunk 1 above)
Comment (mightyguava) [reply README.md:53]: yea oops

File: README.md [hunk 2]
```diff
@@ -46,10 +46,11 @@ You will be prompted to confirm a self-signed certificate is really desired,
 then prompted for the PIN, and then prompted to touch your Yubikey.  
 The output will contain 3 `CERTIFICATE` blocks instead of a `CERTIFICATE_REQUEST` at the end (example output below).  
 **This option is useful mostly for testing purposes.**
+The `--assume-yes` flag can be used in combination with the `--self-sign` option to disable its y/n prompt.
 
 Add the `--no-csr` flag to skip the certificate signing request being printed. In this case, you will not be prompted to touch your Yubikey.  
 This option is useful if you don't need the generated key to be a part of an existing PKI.  
-you can still verify the key's certificate using Yubico's certificate [here](https://developers.yubico.com/PIV/Introduction/PIV_attestation.html)
+you can still verify the key's certificate using Yubico's certificate [here](https://developers.yubico.com/PIV/Introduction/PIV_attestation.html).
```
Comment (yoavamit) [inline README.md:53]: 🤏 😄

**Pull request merged**